components/tooltip/tooltip.js

```diff
@@ -998,7 +998,7 @@ class Tooltip extends RtlMixin(LitElement) {
 			await clone.updateComplete;
 		}
 
-		this._truncating = clone.scrollWidth > target.offsetWidth;
+		this._truncating = (clone.scrollWidth - target.offsetWidth) > 2;
```

Comment:
Could it be related to a border somewhere? Granted, I think browsers are pretty consistent with measurement nowadays?

It might be worth adding a comment in the code for future us.